xapi-xenopsd-simulator.opam

```diff
@@ -0,0 +1,11 @@
+opam-version: "1.2"
+name: xapi-xenopsd-simulator
+maintainer: "xen-api@lists.xen.org"
+authors: "xen-api@lists.xen.org"
+homepage: "https://github.com/xapi-project/xenopsd"
+build: [[ "jbuilder" "build" "-p" name "-j" jobs ]]
+build-test: [[ "jbuilder" "runtest" "-p" name "-j" jobs ]]
```

Comment:
I think we only have tests for the `xapi-xenopsd` package, so we could omit this.

Reply:
Ok, will remove it from this package.
Apropos, there is a test file for xenlight, but at the moment I don't think we run it.